macOSUpgrade.sh

```diff
@@ -373,7 +373,7 @@ fi
 # APPLICATION
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-if ! [[ ${pwrStatus} == "OK" ]] && [[ ${spaceStatus} == "OK" ]]; then
+if [ "$pwrStatus" != "OK" ] || [ "$spaceStatus" != "OK" ]; then
```

Comment:
There was a problem in applying the negative statement.

Reply:
LGTM